transmogrifier/sources/whoas.py

```diff
@@ -0,0 +1,34 @@
+from typing import List
```

Comment:
Totally optional, but this is unnecessary. You can just use the built-in type `list` for type hinting.